apps/chef_db/src/chef_db.erl

```diff
@@ -230,6 +230,11 @@ fetch(ObjectRec, #context{reqid = ReqId}) ->
                       not_found |
                       {error, term()}.
 fetch_multi(RecModule, #context{reqid = ReqId}, QueryName, QueryParams) ->
+    %% FIXME: 
```

Comment:
Not exactly elegant, but what about: binary_to_atom(iolist_to_binary([atom_to_binary(fetch_multi,utf8), "_", QueryName]))?